### DISCUSSION TITLE
"Configuration file location" spent me a lot of time

### DISCUSSION THREAD

**Discussion by hcchengithub** [trungleduc/jupyter_app_launcher#17] [General]
[The doc says:](https://jupyter-app-launcher.readthedocs.io/en/latest/usage.html#configuration-file-location) "By default, jupyter_app_launcher will look for a configuration file in the list of data directories of Jupyter (i.e. the **data section** of the ` jupyter --path ` command). If under one of these paths, the **jupyter_app_launcher**/config.yaml file exists, it will be used."
Very easily mistaken and it doesn't work without any clue. Don't forget : 

1. the data section
2. under the path create sub directory named "jupyter_app_launcher" and have your config.yaml there !

**Comment by trungleduc** [comment]
Thanks for your report, any PR improving the documentation is welcome.

**Comment by trungleduc** [comment]
Fixed in the last release
